projects/kubernetes-sigs/image-builder/packer/ami/additional_files/bootstrap-after.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
+source /etc/eks/logging.sh
 
-KUBE_VIP_IMAGE=$1
```

Comment:
We need to pass the kube-vip image as input so it's not hardcoded in the ami

Reply:
added, thanks!